uvtools/dspec.py

```diff
@@ -359,7 +360,7 @@ def fourier_filter(x, data, wgts, filter_centers, filter_half_widths, suppressio
                         if not filter2d:
                             info = []
                             for i, _d, _w, _a in zip(np.arange(_data.shape[0]).astype(int), _data, _wgts, area):
-                                if _w[0] < skip_wgt:
+                                if _w[0] < skip_wgt or np.all(_d) == 0.:
```

Comment:
I'm not sure whether or not it's only if _d is precisely zero. I suspect it might not be because the halt condition for clean is set by when the RMS resid is below tol x original RMS which is technically only impossible when original RMS = 0. However, I'm also be concerned with underflow so I'm going to use np.isclose.

Reply:
Could you also add a more informative comment in the `fourier_filter` doc string  as to what `max_contiguous_edge_flags` is doing--its not clear to me based on the current description. thanks